### ISSUE TITLE
Crashing on startup/launch

### ISSUE THREAD

**Issue by Vertliz** [AIRLegend/aitrack#190]
When I click the Ai Track.exe it launches the GUI and then it crashes. Please help!

**Comment by searching46dof** [comment]
Please see https://github.com/AIRLegend/aitrack/issues/171 since it appears to be the same symptom

**Comment by FrankSpierings** [comment]
I've noticed that the application will also crash if the user is not able to write in that location. I first placed the binaries in `C:\Program Files\AITRack` when this occured.

This happens because of https://github.com/AIRLegend/aitrack/blob/master/Client/src/Main.cpp#L44, but there is more code writing to the same directory. This type of info should be written to a subdir within `%APPDATA%` as an improvement.

**Comment by searching46dof** [comment]
I believe it saves the configuration to prefs.ini in the same folder so it would require write privilege.

I would prefer prefs.ini remain in the same folder so that I can easily run multiple instances concurrently. I'm working on an application to extend the functionality by averaging the udp stream data for better coverage and more accuracy through parallax.

**Comment by vdruts** [comment]
I'm having this issue except GUI opens but crashes once I select track.

**Comment by searching46dof** [comment]
@bbecausereasonss , crashing when selecting "track" usually indicates a webcam driver issue, a chipset driver issue or incompatible settings for the webcam (width, height, FPS). check the camera manufacturer's website for the the supported settings.

**Comment by Aberro** [comment]
Is it possible to at least have some logs to know what's wrong?
Because this:
[2023-01-13 11:22:16.706] [aitrack] [info] Starting camera 0 capture
is the last line before crash. And it does not explain anything.

**Comment by searching46dof** [comment]
crashes are recorded by windows in the windows event viewer logs.
Check that your login account has full rights to the folder.
Check windows settings that your account has privacy rights to the webcam.
And check if it may be blocked by antivirus. 

also see https://github.com/AIRLegend/aitrack/issues/183 which this may be a duplicate of.
